setup.py

```diff
@@ -15,6 +16,8 @@
 from shutil import copyfile
 
 from setuptools import Extension, setup
+from setuptools.command.install import install as InstallCommandBase
+from wheel.vendored.packaging.tags import sys_tags
 
 nightly_build = False
```

Comment:
Variables used in master. Should we change it to upper case or disable the pylint warning for this file